vassal-app/src/main/java/VASSAL/build/GameModule.java

```diff
@@ -1103,6 +1104,14 @@ protected void save(boolean saveAs) {
 
       lastSavedConfiguration = save;
     }
+    catch (FileSystemException e) {
+      String[] msgs = e.getLocalizedMessage().split("\n");
```

Comment:
`final`

Reply:
Ah yes - it's funny that it doesn't seem to be failing my build on this -- maybe we forgot to actually turn the rule on.

Reply:
It's because PMD is only set up on master... but then this will cause master to fail as soon as it gets merged in to master.